app/components/withdrawal_details/view.rb

```diff
@@ -4,13 +4,18 @@ module WithdrawalDetails
   class View < GovukComponent::Base
     include SummaryHelper
 
-    attr_reader :data_model, :deferred
+    attr_reader :data_model, :deferred, :trainee
 
-    def initialize(data_model)
+    def initialize(data_model, trainee)
       @data_model = data_model
+      @trainee = trainee
       @deferred = data_model.trainee.deferred?
     end
```

Comment:
`trainee` does not need to be a method argument as you can just do `data_model.trainee`
You could add a delegate to trainee if you like.